src/cdk/v2/destinations/zoho/utils.js

```diff
@@ -11,13 +11,16 @@ const { isHttpStatusSuccess } = require('../../../../v0/util');
 const { handleHttpRequest } = require('../../../../adapters/network');
 const { CommonUtils } = require('../../../../util/common');
 
-const deduceModuleInfoV2 = (Config, destConfig) => {
-  const { object, identifierMappings } = destConfig;
+const getRegion = (destination) =>
+  destination?.deliveryAccount?.options?.region || destination?.Config?.region;
```

Comment:
If deliveryAccount contains accountDefinition then check for region inside deliveryAccount otherwise fallback to the old logic. If deliveryAccount exists but not options then log that.

Reply:
As per discussion with @koladilip we should throw retryble error from here.